code/ui/blocks/src/blocks/ArgTypes.tsx

```diff
@@ -70,5 +70,7 @@ export const ArgTypes: FC<ArgTypesProps> = (props) => {
 
   const filteredArgTypes = filterArgTypes(argTypes, include, exclude);
 
+  if ('of' in props && !props.of) throw new Error('Unexpected `of={}` with undefined prop');
```

Comment:
```suggestion
  if ('of' in props && props.of === undefined) throw new Error('Unexpected `of={}` with undefined prop');
```

Is this more correct?

Reply:
In this specific case I agree that it's better to be explicit and check for `undefined`